src/sentry/sentry_metrics/indexer/ratelimiters.py

```diff
@@ -103,23 +103,15 @@ def __exit__(self, exc_type: Any, exc_value: Any, traceback: Any) -> None:
         Consumes the rate limits returned by `check_write_limits`.
         """
         if exc_type is None:
-            self._writes_limiter._get_rate_limiter(self._use_case_id, self._namespace).use_quotas(
+            self._writes_limiter.rate_limiter.use_quotas(
                 self._requests, self._grants, self._timestamp
             )
 
 
 class WritesLimiter:
-    def __init__(self) -> None:
-        self.rate_limiters: MutableMapping[str, RedisSlidingWindowRateLimiter] = {}
-
-    def _get_rate_limiter(
-        self, use_case_id: UseCaseKey, namespace: str
-    ) -> RedisSlidingWindowRateLimiter:
-        if namespace not in self.rate_limiters:
-            options = get_ingest_config(use_case_id).writes_limiter_cluster_options
-            self.rate_limiters[namespace] = RedisSlidingWindowRateLimiter(**options)
-
-        return self.rate_limiters[namespace]
+    def __init__(self, namespace: str, **options: Any) -> None:
```

Comment:
Can we properly type this? It's not particularly clear what `options` are supposed to be here.